gene2phenotype_project/gene2phenotype_app/views/disease.py

```diff
@@ -471,4 +471,105 @@ def post(self, request):
         if errors:
             response_data["error"] = errors
 
-        return Response(response_data, status=status.HTTP_200_OK if updated_records else status.HTTP_400_BAD_REQUEST)
\ No newline at end of file
+        return Response(response_data, status=status.HTTP_200_OK if updated_records else status.HTTP_400_BAD_REQUEST)
+    
+
+class UpdateDiseaseOntologyTerms(BaseAdd):
+    """
```

Comment:
Suggesting we change the docstring to follow the python format

Reply:
You mean I should remove the first emtpy line?

Reply:
Yeah like the standard is 
 """Method to update the term and/or description of Ontology Terms in bulk.

Reply:
I think both ways work: https://peps.python.org/pep-0257/#handling-docstring-indentation

Reply:
Alright then.